README.md

```diff
@@ -25,22 +25,45 @@ The Biconomy SDK is your all-in-one toolkit for building decentralized applicati
 Unlock the full potential of **ERC4337 Account Abstraction** with methods that simplify the creation and dispatch of UserOperations, streamlining dApp development and management.
 
 ```javascript
-const biconomyAccount = new BiconomySmartAccount(biconomySmartAccountConfig);
-const biconomySmartAccount = await biconomyAccount.init();
-console.log("owner: ", biconomySmartAccount.owner);
-console.log("address: ", biconomySmartAccount.address);
+
+import { ECDSAOwnershipValidationModule, DEFAULT_ECDSA_OWNERSHIP_MODULE } from "@biconomy/modules";
+import { IBundler, Bundler } from '@biconomy/bundler'
+import { DEFAULT_ENTRYPOINT_ADDRESS } from "@biconomy/account"
+import { providers } from 'ethers'
+import { ChainId } from "@biconomy/core-types"
+
+
+const module = await ECDSAOwnershipValidationModule.create({
+  signer: wallet,
+  moduleAddress: DEFAULT_ECDSA_OWNERSHIP_MODULE
+  })
+
+const biconomySmartAccount = await BiconomySmartAccountV2.create({
+    chainId: ChainId.POLYGON_MUMBAI,
+    bundler: bundler,
+    paymaster: paymaster, 
+    entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
+    defaultValidationModule: module,
+    activeValidationModule: module
+})
+
+console.log("address: ", await biconomySmartAccount.getAccountAddress());
 ```
 
 ### Bundler
 
 Leverage standardized bundler infrastructure for efficient operation of account abstraction across EVM networks.
 
 ```javascript
+
+import { IBundler, Bundler } from '@biconomy/bundler'
+
+
 const bundler: IBundler = new Bundler({
-    bundlerUrl: '', // From Biconomy Dashboard
+    bundlerUrl: 'https://bundler.biconomy.io/api/v2/80001/nJPK7B3ru.dd7f7861-190d-41bd-af80-6877f74b8f44',     
```

Comment:
Instead of this can add `<bundlerKey>` and a comment, `// Please go to https://dashboard.biconomy.io and generate bundler`.

Reply:
done